packages/auth/src/http-auth-repo.ts

```diff
@@ -38,6 +40,9 @@ export class HttpAuthRepo implements AuthRepo {
     invariant(response.status === 200, 'Could not get auth user');
```

Comment:
➡️ Proposal | Call `onUserAuthenticatedCallback(null)` if the user is not logged